docs/teal-basics.md

```diff
@@ -0,0 +1,121 @@
+
+Instead, they’re characterized by features like **self-managed teams**, intuitive reasoning and **decentralized decision-making**. Because these practices are so new, the conversation around what they are and how to implement them is **ongoing**.
+
+## What is a process ?
+
+A simple Wikipedia entry can help us understand what a process is:
+
+> “A business process or business method is a collection of related, structured activities or tasks by people or equipment in which a specific sequence produces a service or product (serves a particular business goal) for a particular customer or customers.”
+
+We created an evaluation page to make sure we & you know what a good process is all about:
+
+- [What is a valid submission to teal.garden?](https://hackmd.io/dMlEYrGOTIm8Yw4e50V8qA?view)
+
+## What is a component ?
+
+The components are the building blocks of our process. Most of the time, a simple list is sufficient, to make sure we have all requirements in place.
+
+A component can be **a role, a policy, or a tool**. They can be exisiting in the organizations setup, or will be added during the process. All these preparing investigations help understand the dynamics of the components and applying them in a new setup.
+
+**Roles & Policies are connected to People.
+Tools are connected to Technology.**
+
+Anyone who wants to adopt your process will need to follow the instructions to setup a tool. It consist of one or several steps. Each step decribes how to integrate one tool into the process. Make sure to document each component which is related to a tool.
+
+## What is a adoption requirement ?
+
+While collecting ideas and checking the components, ask yourself the following questions:
+
+- What is essential to perform this process?
+- Which prerequisites are needed to adopt this process as practice?
+
+**Examples:**
+
+- "The organization needs to practice holacracy v4.1 or higher"
+
+or:
+
+- "Each team-member needs a web-3 enabled wallet."
+
+## What is a Role ?
+
+### without context of holacracy:
+
+A "role" as the set of expectations that society places on an individual. By unspoken consensus, certain behaviors are deemed "appropriate" and others "inappropriate".
+
+**Example:**
+
+> "An "appropriate" doctor dresses fairly conservatively, asks a series of personal questions about one's health, touches one in ways that would normally be forbidden, writes prescriptions, and shows more concern for the personal wellbeing of his or her clients than is expected of, say, an electrician or a shopkeeper."
+
+Roles are not limited to occupational status, of course, nor does the fact that one is cast in the role of "doctor" during working hours prevent one from taking on other roles at other times: husband/wife, friend, father/mother, and so on.
+
+**-- Roles are the expectations of society placed on the individual.**
+
+### in context of holacracy:
```

Comment:
```suggestion
### in context of teal organizations:
```

Reply:
maybe we put the docs more in the light of teal organizations? because holacracy is kind of special.